geoapps/inversion/line_sweep/driver.py

```diff
@@ -116,19 +117,31 @@ def line_files(path: str | Path):
     def collect_results(self):
         path = Path(self.workspace.h5file).parent
         files = LineSweepDriver.line_files(str(path))
-        lines = np.unique(self.pseudo3d_params.line_object.values)
-        data_result = self.pseudo3d_params.data_object.copy(
-            parent=self.pseudo3d_params.out_group
-        )
-
+        line_ids = self.pseudo3d_params.line_object.values
         data = {}
         drape_models = []
-        for line in lines:
+        for line in np.unique(line_ids):
             with Workspace(f"{path / files[line]}.ui.geoh5") as ws:
-                survey = ws.get_entity("Data")[0]
-                data = self.collect_line_data(survey, data)
-
-                mesh = ws.get_entity("Models")[0]
+                out_group = [
```

Comment:
what if there is more than one SimPEGGroup?  Is there a ga_group_name we can use here?

Reply:
I see your point, but this is technically all internal mechanics of LineSweeper, so we already know that there is only going to be one group in there. We are not changing the name of that group as far as I can tell.